requirements/test.txt

```diff
@@ -1,6 +1,5 @@
 -r base.txt
```

Comment:
Why did you rename the file? Aren't this the dev requirements? I think dev requirements should include the test requirements.

Reply:
Projects requirements are in setup.py itself now. To install them run `pip install -e .`.
And `requirements/test.txt` are only used to run tests. that why I renamed it.